### PR TITLE
UC Mobile cheatsheet

### PR DESCRIPTION
**Pull request by gautamkrishnar**
Continuing #2137 

---
IA Page: https://duck.co/ia/view/uc_browser_mobile_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  },
+  "template_type": "keyboard",
+  "section_order": [
+    "shortcuts"
+  ],
+  "sections": {
+    "shortcuts": [
+      {
+        "val": "Main Shortcuts",
+        "key": "[#]  [ 1 ]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: @gautamkrishnar let's remove the spaces around the numbers: `[ 1 ] -> [1]`

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      },
+      {
+        "val": "Background Tab open",
+        "key": "[#] , [1]  /  [ 11 ] "
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: Lets remove the space before the `,`

`"key": "[#] , [1]  /  [ 11 ] "` -> `"key": "[#], [1] / [11] "`

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
+{
+  "id": "uc_browser_mobile_cheat_sheet",
+  "name": "UC Browser Mobile",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: Not sure if we really need to specify `Mobile`? Isn't UC only a mobile browser?

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      {
+        "val": "Close one Tab.",
+        "key": "[#] , [6]  /  [ 12 ]"
+      },
+      {
+        "val": " Add Bookmark.",
+        "key": "[#] , [4]  /  [ 15 ]"
+      },
+      {
+        "val": "Page up / Move Left.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: Let's remove the trailing periods and make sure each of these keys are wrapped in `[]`

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+      },
+      {
+        "val": "Quick Search (17)",
+        "key": "[#] , [5]"
+      },
+      {
+        "val": "Switch Themes",
+        "key": "[#] , [*]"
+      },
+      {}
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: this is breaking the cheat sheet!

**Comment by gautamkrishnar** [comment]
@moollaza done. Pls check it out

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json (hunk 3 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: yes this keyboard shortcut is specific to mobile version of ucbrowser. (The java version of UC)

I am not sure whether we have to include this in our search engine. There's not much java phones right now!

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+  "id": "uc_browser_mobile_cheat_sheet",
+  "name": "UC Browser Mobile",
+  "description": "Keyboard shortcuts for UC Browser Mobile",
+  "metadata": {
+    "sourceName": "Ucweb",
+    "sourceUrl": "http://www.ucweb.com/a/ppress/2012/1211/2628.htmlhttp://www.ucweb.com/a/ppress/2012/1211/2628.html"
+  },
+  "template_type": "keyboard",
+  "section_order": [
+    "shortcuts"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: Lets use `Shortcuts`.

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+      {
+        "val": "Switch Next  Window",
+        "key": "3"
+      },
+      {
+        "val": "Reload (Refresh)",
+        "key": "*"
+      },
+      {
+        "val": " Go Forward",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: Rogue space here - might want to check the rest of the items as well.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=uc browser mobile cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/uc_browser_mobile_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+        "val": "Go Forward",
+        "key": "9"
+      },
+      {
+        "val": "Go Backward",
+        "key": "7"
+      },
+      {
+        "val": "Copy",
+        "key": "[1],[3]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: @gautamkrishnar you need a space after the `,` for all of the below `key`s

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+  ],
+  "sections": {
+    "Shortcuts": [
+      {
+        "val": "Main Shortcuts",
+        "key": "[#]  [1]"
+      },
+      {
+        "val": "Background Tab open",
+        "key": "[#], [1]  /  [11] "
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: Trailing Space here

**Comment by jdorweiler** [comment]
@gautamkrishnar just a few things to fix and it is ready to go.

**Comment by gautamkrishnar** [comment]
@moollaza done :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=uc browser mobile cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/uc_browser_mobile_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 10]
```diff
@@ -0,0 +1,93 @@
+{
+  "id": "uc_browser_mobile_cheat_sheet",
+  "name": "UC Browser Mobile",
+  "description": "Keyboard shortcuts for UC Browser Mobile",
+  "metadata": {
+    "sourceName": "Ucweb",
+    "sourceUrl": "http://www.ucweb.com/a/ppress/2012/1211/2628.htmlhttp://www.ucweb.com/a/ppress/2012/1211/2628.html"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: This URL seems a bit messy! Looks like you might have several strung together!

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+      {
+        "val": "Free Copy (131)",
+        "key": "[#] , [8]"
+      },
+      {
+        "val": "Saved Bookmarks.",
+        "key": "[#] , [2]"
+      },
+      {
+        "val": "Go Previous Tab.",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: Lets remove periods after the short descriptions.

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json (hunk 10 above)
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: oops :wink:

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 12]
```diff
@@ -0,0 +1,93 @@
+{
+  "id": "uc_browser_mobile_cheat_sheet",
+  "name": "UC Browser Mobile",
+  "description": "Keyboard shortcuts for UC Browser Mobile",
+  "metadata": {
+    "sourceName": "Ucweb",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: `UCWeb`

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=uc browser mobile cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/uc_browser_mobile_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/uc-browser-mobile.json [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+        "val": "Background Tab open",
+        "key": "[#], [1]  /  [11]"
+      },
+      {
+        "val": "Close one Tab",
+        "key": "[#], [6]  /  [12]"
+      },
+      {
+        "val": "Add Bookmark",
+        "key": "[#], [4]  /  [15]"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/uc-browser-mobile.json]: [We should wrap alternative keys with parentheses](http://docs.duckduckhack.com/frontend-reference/cheat-sheet-reference.html#alternative-keys-eg-pressing-either-a-or-b).

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=uc browser mobile cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/uc_browser_mobile_cheat_sheet_mobile.png)

**Comment by GuiltyDolphin** [comment]
@gautamkrishnar Thanks! :dolphin:

**Pull request merged**